### PR TITLE
Tino's card

### PR DESCRIPTION
**Pull request by TinoTheGiant**
Add my card information

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -121,6 +121,33 @@ <h2>A project for first-time contributions</h2>
...
+         <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+         <div class="card">
+          <p class="name"> Tino Kaseke</p>
+          <p class="contact">
+            <i class="fa-brands fa-bluesky"></i>
+            <a href="https://bsky.app/profile/tinodevelops.bsky.social" target="_blank">@Tino_TheGiant</a>
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/TinoTheGiant" target="_blank">TinoTheGiant</a>
+          </p>
+          <p class="about">Full stack developer| Data Analyst. Lets collab! I enjoy hyrox and crossfit.</p>
```
Comment (StackOverflowIsBetterThanAnyAI) [inline index.html:133]: typo
```suggestion
          <p class="about">Full stack developer | Data Analyst. Lets collab! I enjoy hyrox and crossfit.</p>
```

File: index.html [hunk 2]
```diff
@@ -121,6 +121,33 @@ <h2>A project for first-time contributions</h2>
...
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://fonts.google.com/" target="_blank" title="First Resource">fonts.google.com</a>
```
Comment (StackOverflowIsBetterThanAnyAI) [inline index.html:138]:
```suggestion
                <a href="https://fonts.google.com/" target="_blank" title="First Resource">Google Fonts</a>
```

File: index.html [hunk 3]
```diff
@@ -121,6 +121,33 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="https://learn.microsoft.com/en-us/visualstudio/python/?view=vs-2022" target="_blank" title="Second Resource">learn.microsoft.com</a>
```
Comment (StackOverflowIsBetterThanAnyAI) [inline index.html:141]:
```suggestion
                <a href="https://learn.microsoft.com/en-us/visualstudio/python/?view=vs-2022" target="_blank" title="Second Resource">Microsoft Learn</a>
```

File: index.html [hunk 4]
```diff
@@ -121,6 +121,33 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="https://skillicons.dev/" target="_blank" title="Third resource">skillicons.dev/</a>
```
Comment (StackOverflowIsBetterThanAnyAI) [inline index.html:144]:
```suggestion
                <a href="https://skillicons.dev/" target="_blank" title="Third resource">Skill Icons/</a>
```

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@TinoTheGiant, Congratulations on your PR Merge, and Thanks for your Contribution. You can check your card on our [Live Website](https://syknapse.github.io/Contribute-To-This-Project)! It may take 2-3 minutes to make your card visible on our website. 

Welcome to the Open Source Community. You can follow the [Next Steps](https://github.com/Syknapse/Contribute-To-This-Project#next-steps) now. You are also welcome to join our [Discord Server: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF) and be a part of our community.

**Pull request merged**